automation/simulator/deck.py

```diff
@@ -187,3 +187,34 @@ def check(self, TC: Card, TR: int, mod: int = 0):
                 result += "Color "
             result += "Hit" if draw.val in TC.range(TR) else "Miss"
         logger.info(f"Drew {draw} vs {TC} with TR {TR}: {result}")
+
```

Comment:
Please don't add things I send you in chat unless you've changed it in some way. If we merge the changes here, I then need to resolve conflicts on my fork

Reply:
Sorry, I can attempt to remove these from my file and re-commit to remove them from the PR.

Reply:
thanks!